### PR TITLE
Query and update animals

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
> Add a column species of type string to your animal's table In the schema file
> Insert the required data in the data file.
>Inside a transaction updates the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
 > Query the complicated queries as required.

> Answer the query questions.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+INSERT INTO animals VALUES (DEFAULT, 'Plantmon', '2021-11-15', 3, '0', -5.7);
+INSERT INTO animals VALUES (DEFAULT, 'Squirtle', '1993-04-02', 1, '1', -12.13);
+INSERT INTO animals VALUES (DEFAULT, 'Angemon', '2005-06-12', 1, '1', -45.13);
+INSERT INTO animals VALUES (DEFAULT, 'Boarmon', '2005-06-07', 7, '1', 20.4);
+INSERT INTO animals VALUES (DEFAULT, 'Blossom', '1998-10-13', 3, '1', 17);
+INSERT INTO animals VALUES (DEFAULT, 'Ditto', '2022-05-14', 4, '1', 22);
+
+-- Add species to the table
+ALTER TABLE animals
+ADD species varchar(20)
```
Comment (mikethreels) [inline data.sql]: - [ ] Everything that creates or change the tables structure should be saved in the schema.sql file. this means that these lines should be in the schema.sql file.

File: queries.sql [hunk 2]
```diff
@@ -35,4 +35,43 @@ where name <> 'Gabumon';
...
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT name, species from animals;
+
+-- delete the records
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql]: - [ ] you should check how many lines there are in the animals table after deleting as well before the ROLLBACK

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
- [x] you should add screenshots of the query results to the pull request description.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @eng-mohamed-eldeeb , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**